bigtop-manager-server/src/main/resources/application.yml

```diff
@@ -30,10 +30,10 @@ spring:
   main:
     banner-mode: log
   datasource:
-    driver-class-name: org.postgresql.Driver
-    url: jdbc:postgresql://localhost:5432/bigtop_manager
-    username: postgres
-    password: postgres
+    driver-class-name: com.mysql.cj.jdbc.Driver
+    url: jdbc:mysql://localhost:3306/bigtop_manager
+    username: root
+    password: root
```

Comment:
@Cat-Drink Please withdraw the submission of this code, there is no need to modify it here